git-try-push/main.js

```diff
@@ -47,7 +47,14 @@ async function main() {
                 // and try again.
                 const delay = (i+1)**2
                 await exec.exec("sleep", [delay])
-                await exec.exec("git", ["pull", "--rebase", "--autostash", remote, branch])
+                // `git pull` can also fail, so do the same retry procedure here.
+                for (let j = 0; j < tries; j++) {
```

Comment:
Does this need another loop? Can it just defer to the previous loop?

Reply:
Not sure what you mean here. We only pull after push fails. Since pull can fail too, it needs to also have its retry loop, and therefore its own try/catch.